sway-book/src/code/language/program-types/scripts/simple/src/main.sw

```diff
@@ -0,0 +1,6 @@
+script;
+
+// All scripts require a main function. The return type is optional.
+fn main(amount: u64) -> bool {
+    true
+}
```

Comment:
An optional suggestion: what do you think about including a 'realistic' short script/predicate within the examples? As a new Sway developer this could give some context into what would be a useful script/predicate to write, instead of having an fn main that just returns true.